mix.exs

```diff
@@ -45,7 +45,7 @@ defmodule EctoInterval.Mixfile do
       {:ex_doc, ">= 0.0.0", only: :dev},
       {:postgrex, "~> 0.14.0 or ~> 0.15.0", optional: true},
       {:ecto, "~> 3.0", optional: true},
-      {:phoenix_html, "~> 2.12", optional: true}
+      {:phoenix_html, "~> 3.0", optional: true}
```

Comment:
Could you please change the phoenix_html required version to `"~> 2.12 or ~> 3.0"` to keep support with legacy projects?

Current projects probably have `{:ecto_interval, "~> 0.2.2"}` in `mix.exs` so automatic upgrade will effectively break their builds.